packages/web-components/src/components/va-accordion/va-accordion-item.scss

```diff
@@ -1,5 +1,56 @@
+@forward 'settings';
+
+@use 'usa-accordion/src/styles/usa-accordion';
+
+@import '../../global/formation_overrides';
+
 @import '../../mixins/focusable.css';
 
+
+
+.usa-accordion__header, .usa-accordion__subheader {
+  display: block;
+}
+
+.usa-accordion__header ::slotted([slot=icon]), .usa-accordion__subheader ::slotted([slot=subheader-icon]){
+  margin-right: 15px;
+  width: inherit;
+}
+
+.usa-accordion__subheader {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 26px;
+  margin-top: 2px;
+}
+
+.usa-accordion__button {
+  position: relative;
+}
```

Comment:
Are these USWDS style overrides?

```
.usa-accordion__header, .usa-accordion__subheader {
  display: block;
}

.usa-accordion__header ::slotted([slot=icon]), .usa-accordion__subheader ::slotted([slot=subheader-icon]){
  margin-right: 15px;
  width: inherit;
}

.usa-accordion__subheader {
  font-weight: 400;
  font-size: 16px;
  line-height: 26px;
  margin-top: 2px;
}

.usa-accordion__button {
  position: relative;
}
```

Reply:
The uswds styles do not include a subheader and icons for the header and subheader lines. These are the styles to add them.

Reply:
ah! Ok so I guess what is confusing is that it's using the `.usa-accordion` namespace which makes me think it's actually from the USWDS module. But it's really just custom classes we added for styling features that are VA specific.

This might be the first time we've done something like this (add a new feature to the USWDS variation - subheader and icons in this case) so it's an interesting scenario we will probably come across again in other components.

Will mimicking a USWDS class for custom styling make it more or less confusing when we eventually need to reconcile a component for the USWDS team? @Andrew565 Considering you've done some preliminary work for "giving back" these components, what are your thoughts on how we should do this? Would it be easier to identify our custom features by using generic classes, or, create best-guess names that are similar to what's in the modules (but not actually in the modules)? Do we even know if that team wants any of our "enhancements"?